_posts/0000-01-02-Sadiq-Shaik.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Goal for Manchester United!
![arsenal v manchester united uefa champions eepjiw0lmqat](https://user-images.githubusercontent.com/48231692/53930631-05625000-40b8-11e9-9d4b-5a0e6ac6a4d0.jpg)